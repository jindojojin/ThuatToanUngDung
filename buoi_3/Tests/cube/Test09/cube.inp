1 60 60
