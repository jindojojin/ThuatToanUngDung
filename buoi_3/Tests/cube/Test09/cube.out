0 0 3364 232 4 0 0
